### PR TITLE
Update rubygem-foreman_ansible_core to 2.1.2

### PR DESCRIPTION
**Pull request by bastilian**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.19
* [ ] 1.18
* [ ] 1.17

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

**Comment by ares** [comment]
I think you should drop these lines
```
Requires: %{?scl_prefix}rubygem(net-ssh) >= 4.0
Requires: %{?scl_prefix}rubygem(net-ssh) < 5.0
```

there's a nice gem plugin to compare differences between versions,
`gem compare foreman_ansible_core 2.1.1 2.1.2`
tells you
```
  DIFFERENT runtime dependencies:
    2.1.1->2.1.2:
      * Updated:
            net-ssh from: ["~> 4.0"] to: [">= 0"]
```

so you know, dependencies in spec must be changed

**Comment by ares** [comment]
LGTM now, this should now restore the same versions for both debs and rpms, deferring for proper review by packaging team

**Comment by bastilian** [comment]
Perfect! Thanks for taking a look, @ares!

File: packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec [hunk 1]
```diff
@@ -18,8 +18,6 @@ Requires: %{?scl_prefix}rubygem(foreman-tasks-core) >= 0.1
 Requires: %{?scl_prefix}rubygem(foreman-tasks-core) < 1.0
 Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) >= 1.1
 Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) < 2.0
-Requires: %{?scl_prefix}rubygem(net-ssh) >= 4.0
```
Comment (ekohl) [inline packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: It shouldn't fully remove it, it's still a dependency. It's only the version constraint that has been removed.

Comment (ares) [reply packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: oh sure, so just keeping one line `Requires: %{?scl_prefix}rubygem(net-ssh)`

Comment (ekohl) [reply packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: Correct

File: packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec [hunk 2]
```diff
@@ -19,7 +19,6 @@ Requires: %{?scl_prefix}rubygem(foreman-tasks-core) < 1.0
 Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) >= 1.1
 Requires: %{?scl_prefix}rubygem(foreman_remote_execution_core) < 2.0
 Requires: %{?scl_prefix}rubygem(net-ssh) >= 4.0
```
Comment (ares) [inline packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec]: this still lists >= 4, I think the part `>= 4.0` should be removed, that sould keep the dependency without version restriction

**Pull request merged**